packages/client/test/integration/Permissions.test.ts

```diff
@@ -0,0 +1,181 @@
+import { Wallet } from 'ethers'
+
+import { clientOptions, createRelativeTestStreamId, fetchPrivateKeyWithGas } from '../test-utils/utils'
+import { StreamrClient } from '../../src/StreamrClient'
+import { Stream } from '../../src/Stream'
+import { StreamPermission } from '../../src/permission'
+import { randomEthereumAddress } from 'streamr-test-utils'
+import { EthereumAddress } from 'streamr-client-protocol'
+import { StreamEndpointsCached } from '../../src/StreamEndpointsCached'
+
+jest.setTimeout(40000)
+
+describe('Stream permissions', () => {
+
+    let client: StreamrClient
+    let stream: Stream
+    let otherUser: EthereumAddress
+
+    beforeAll(async () => {
+        const wallet = new Wallet(await fetchPrivateKeyWithGas())
+        otherUser = randomEthereumAddress()
+        client = new StreamrClient({
+            ...clientOptions,
+            auth: {
+                privateKey: wallet.privateKey,
+            }
+        })
+    })
```

Comment:
Add cleanup (even thought its not strictly needed here since connections aren't formed)
```ts
afterAll(async () => {
    await client?.destroy()
})
```

Reply:
Fixed